scripts/build/build.js

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const path = require("path");
+const shell = require("shelljs");
+const pkg = require("../../package.json");
+
+const rootDir = path.join(__dirname, "..", "..");
+const parsers = [
+  "babylon",
+  "flow",
+  "typescript",
+  "graphql",
+  "postcss",
+  "parse5",
+  "json"
+];
+
+process.env.PATH += path.delimiter + path.join(rootDir, "node_modules", ".bin");
+
+function pipe(string) {
+  return new shell.ShellString(string);
+}
+
+shell.set("-e");
+shell.cd(rootDir);
+
+shell.rm("-Rf", "dist/", "docs/lib/");
+shell.mkdir("-p", "docs/lib/");
+
+// --- Lib ---
+
+shell.echo("Bundling lib index...");
+shell.exec("rollup -c scripts/build/rollup.index.config.js");
+
+shell.echo("Bundling lib bin...");
+shell.exec("rollup -c scripts/build/rollup.bin.config.js");
+shell.chmod("+x", "./dist/bin/prettier.js");
+
+for (const parser of parsers) {
+  if (parser === "postcss") {
+    continue;
+  }
+  shell.echo(`Bundling lib ${parser}...`);
+  shell.exec(
+    `rollup -c scripts/build/rollup.parser.config.js --environment parser:${parser}`
+  );
+}
+
+shell.echo("Bundling lib postcss...");
+// PostCSS has dependency cycles and won't work correctly with rollup :(
+shell.exec(
+  "webpack --hide-modules src/parser-postcss.js dist/parser-postcss.js"
+);
+// Prepend module.exports =
+const content = shell.cat("dist/parser-postcss.js").stdout;
+pipe(`module.exports = ${content}`).to("dist/parser-postcss.js");
+
+shell.echo();
+
+// --- Docs ---
+
+shell.echo("Bundling docs index...");
+shell.cp("dist/index.js", "docs/lib/index.js");
+shell.exec(
+  "node_modules/babel-cli/bin/babel.js dist/index.js --out-file docs/lib/index.js --presets=es2015"
+);
+
+shell.echo("Bundling docs babylon...");
+shell.exec(
+  "rollup -c scripts/build/rollup.docs.config.js --environment filepath:parser-babylon.js"
+);
+shell.exec(
+  "node_modules/babel-cli/bin/babel.js docs/lib/parser-babylon.js --out-file docs/lib/parser-babylon.js --presets=es2015"
+);
+
+for (const parser of parsers) {
+  if (parser === "babylon") {
+    continue;
+  }
+  shell.echo(`Bundling docs ${parser}...`);
+  shell.exec(
+    `rollup -c scripts/build/rollup.docs.config.js --environment filepath:parser-${parser}.js`
+  );
+}
+
+shell.echo();
+
+// --- Misc ---
+
+shell.echo("Remove eval");
+shell.sed(
+  "-i",
```

Comment:
The point of `shelljs` is that it _looks_ like bash, but doesn't use any shell script at all. It's just JavaScript / Node.js APIs behind the scenes. https://github.com/shelljs/shelljs/blob/38b57c8942e68a8bff35fb9177a856dd6f62222a/src/sed.js#L79-L81